DEV-4/src/Main.java

```diff
@@ -0,0 +1,44 @@
+import counters.FrequencyCounter;
+import counters.WordsCounter;
+import outputer.Outputer;
+import stringHandlers.StringConcatenator;
+import stringHandlers.StringParser;
+import stringHandlers.StringReader;
+
+/**
+ * Main class in application where others classes are gathering
+ */
+public class Main {
+  
+  private static final INTERRUPT_NUMBER_OF_OPERATIONS = 3;
+
+  /**
+   * Entry point in application
+   * uses infinity cycle
+   * after 3 iterations could be interrupted
+   */
+  public static void main(String[] args) {
+    int operationCounter = 0;
+    StringConcatenator concatenator = new StringConcatenator();
+    StringReader reader = new StringReader();
+    Outputer outputer = new Outputer();
+    String receivedText;
+
+    while (true) {
```

Comment:
i wanted get opportunity to user print more text ad lib. but at least 3 times and then he could break the loop